README.md

```diff
@@ -1,94 +1,96 @@
-  WARRANTY; without even the implied warranty of MERCHANTABILITY or
-  FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
-  for more details.
+Copyright (C) 1997-2016 Adrian Sandu, Michigan Tech. & Virginia Tech. (USA), with contributions from Rolf Sander, Max-Planck Institute for Chemistry, Mainz (Germany).
 
-  A copy of the GNU General Public License could be found below; you
-  could also consult http://www.gnu.org/copyleft/gpl.html or write to the
-  Free Software Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA
-  02111-1307, USA.
+KPP is free software; you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation (http://www.gnu.org/copyleft/gpl.html); either
+version 2 of the License, or (at your option) any later version.
 
-  Adrian Sandu  
-  Computer Science Department  
-  Virginia Polytechnic Institute and State University  
-  Blacksburg, VA 24060  
-  E-mail: sandu@cs.vt.edu
+KPP is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
 
-******************************************************************************
+A copy of the GNU General Public License could be found below; you
+could also consult http://www.gnu.org/copyleft/gpl.html or write to the
+Free Software Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA
+02111-1307, USA.
 
+Adrian Sandu
+Computer Science Department
+Virginia Polytechnic Institute and State University
+Blacksburg, VA 24060
+E-mail: sandu@cs.vt.edu
 
 ******************************************************************************
 
-  KPP - symbolic chemistry Kinetics PreProcessor  
-        (for version, see src/gdata.h)  
-        http://www.cs.vt.edu/~asandu/Software/Kpp  
-  KPP is distributed under GPL, the general public licence  
-        (http://www.gnu.org/copyleft/gpl.html)   
-    (C) 1995-1997, V. Damian & A. Sandu, CGRER, Univ. Iowa  
-    (C) 1997-2016, A. Sandu, Michigan Tech, Virginia Tech  
-        with contributions from:  
-        R. Sander, Max-Planck Institute for Chemistry, Mainz, Germany
+__KPP__ is a symbolic chemistry Kinetics PreProcessor, developed by
+V. Damian and A. Sandu with contribution by R. Sander
+(https://people.cs.vt.edu/~asandu/Software/Kpp).
+
+__KPP-Mistra__ is a custom version of KPP for the Mistra model
+(https://github.com/MistraModel/Mistra). It was developed starting
+from KPP v2.2.3 and includes all the bugfixes and modifications by
+R. Sander and J. Bock (https://bitbucket.org/gcst/kpp), plus some
+additional modifications specific to Mistra by J. Bock. See
+`CHANGELOG.md` for a summary of the changes.
+
+The stable version of KPP-Mistra can be downloaded here:
+https://github.com/MistraModel/KPP-Mistra/releases.
```

Comment:
```suggestion
```
Needs to be deleted, if changed above. (I just didn't know how to review 2 lines at once.)